crates/wasmtime/src/ref.rs

```diff
@@ -36,6 +36,11 @@ impl ExternRef {
         &*self.inner
     }
 
+    /// Get the reference count for this `ExternRef`.
+    pub fn get_reference_count(&self) -> usize {
```

Comment:
Perhaps we should take a leaf out of `Rc`'s book and use [`strong_count`](https://doc.rust-lang.org/std/rc/struct.Rc.html#method.strong_count) for forwards-compatibility with weak references if we ever add them?